### ISSUE TITLE
Mode detection broken after updating auto-dark-mode.nvim

### ISSUE THREAD

**Issue by adonig** [f-person/auto-dark-mode.nvim#40]
I updated auto-dark-mode.nvim to the latest commits, and now it no longer functions as expected. Here’s the behavior I’ve observed:

- On startup, it always prints set_dark_mode, even when my system is in light mode.
- When switching modes, the expected set_light_mode or set_dark_mode print statements don’t appear.

Before the update, it correctly detected and switched between light and dark modes, but now it seems to default to dark mode on startup, and the mode switching is broken.

Here’s my current configuration:

```lua
return {
  {
    "f-person/auto-dark-mode.nvim",
    lazy = false,
    opts = {
      update_interval = 250,
      set_dark_mode = function()
        print "set_dark_mode"
        local opts = require("nvconfig").base46
        if opts.theme ~= opts.theme_toggle[1] then
          require("base46").toggle_theme()
        end
      end,
      set_light_mode = function()
        print "set_light_mode"
        local opts = require("nvconfig").base46
        if opts.theme ~= opts.theme_toggle[2] then
          require("base46").toggle_theme()
        end
      end,
    },
  },
}
```

Please let me know if there’s something I need to adjust in my configuration or if this is a bug introduced with the latest update.

**Environment:**
- Fedora Workstation 40
- Gnome 46.5
- Nvim v0.10.1

**Comment by dyskette** [comment]
Yeah, dbus-send returns 0 (no preference) in GNOME by default, I don't know about the other desktops but GNOME defaults to light in that case, however this plugin now defaults to dark instead.

It looks like the change was introduced as part of a new `fallback` option, so you can add it inside the `opts` property.

```lua
return {
    "f-person/auto-dark-mode.nvim",
    lazy = false,
    opts = {
        ...
        fallback = "light"
    }
}
```

**Comment by McAllaster** [comment]
Thanks for that solution, `fallback = "light"` works for me on Fedora 40 w/ Gnome! Might be nice to include that in the config docs, though. (Unless I'm blind and it's there already?)

**Comment by nekowinston** [comment]
Thanks for the workaround, the fallback option isn't documented yet, there's a PR open for it (#32).
I hope I can finally make some time tomorrow and finish my two PRs 😅

**Comment by nekowinston** [comment]
I'll close this since #32 was merged a while ago. 🙂
